ironfish-cli/src/commands/accounts/repair.ts

```diff
@@ -109,7 +110,8 @@ export default class Repair extends IronfishCommand {
     walletDb: WalletDB,
     chain: Blockchain,
   ): Promise<void> {
-    let unconfirmedBalance = 0n
+    const unconfirmedBalances = new BufferMap<bigint>()
+    const unconfirmedBalance = 0n
```

Comment:
I don't think we need this variable anymore - it's only used for the log, which probably needs to be updated too

Reply:
Ah yeah forgot to clean it up, thanks